massa-client/src/cmds.rs

```diff
@@ -185,27 +188,35 @@ pub enum Command {
     when_moon,
 }
 
+/// Display the help of all commands
 pub(crate) fn help() {
     println!("HELP of Massa client (list of available commands):");
     Command::iter().map(|c| c.help()).collect()
 }
 
+/// bail a shinny rpc error
 macro_rules! rpc_error {
     ($e:expr) => {
         bail!("check if your node is running: {}", $e)
     };
 }
 
+/// print a yellow warning
 macro_rules! client_warning {
     ($e:expr) => {
         println!("{}: {}", style("WARNING").yellow(), $e)
     };
 }
 
-#[derive(Serialize)]
+/// Used to have a shinny json output
+/// Really I'm not sure why this exist
```

Comment:
We must sort this out before merging. We can have this type of comment in our codebase.